bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/BagsResult.scala

```diff
@@ -0,0 +1,68 @@
+package edu.gemini.spModel.target.env
+
+import edu.gemini.shared.util.immutable.{Option => GOption}
+import edu.gemini.spModel.pio.{Pio, PioFactory, ParamSet}
+import edu.gemini.spModel.target.SPTarget
+import edu.gemini.spModel.rich.shared.immutable._
+
+sealed trait BagsResult extends Cloneable {
+  val id: String
+  val target: Option[SPTarget] = None
+  def targetAsJava: GOption[SPTarget] = target.asGeminiOpt
+  override def clone: BagsResult = this
+
+  override def equals(obj: scala.Any): Boolean = obj match {
+    case other: BagsResult => target.equals(other.target)
```

Comment:
Hmmm... now that you mention it, my guess is that it will, but I could not quite remember how equality is implemented.

I'm also not sure if, as this progresses, we'll want to compare the SPTargets or the underlying ITargets to avoid comparing watchers, but I'm going to take it out completely for now until the need to do something fancier than the default becomes apparent.